### PR TITLE
Url checks

### PR DESCRIPTION
**Pull request by ehudyonasi**
Currently will only check Github, because Linkedin is blocking requests:
[link for explanation](https://stackoverflow.com/questions/18704429/python-requests-library-added-an-additional-header-accept-encoding-identity)

### CODE REVIEW

**Comment by szabgab** [comment]
Thanks for this work, but let's start by removing the .json file from this pr. Then please update the code to reflect the most recent version of generate.py

File: test_json.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+from generate import check_url_for_participant
+
+GITHUB_URL: str = "https://github.com/"
 
 def test_json():
     for folder in ['participants', 'mentors']:
         people = read_json_files(folder)
         for person in people:
             assert 'name' in person
+            assert 'linkedin' in person
```
Comment (szabgab) [inline test_json.py:19]: I don't want to require linkedin. Some people don't have it.

**Comment by szabgab** [comment]
It seems that this PR now contains a lot of changes that were already added to the original repository. It needs to be cleaned up before I can merge it. At this point it might be easier to start from scratch. Remove the fork you have. Create a new fork and apply the changes there.

**Comment by ehudyonasi** [comment]
you are right, I don't know it has all the previous changes in it.. will close this PR and create new fork then.

**Pull request closed without merging**